tests/test_ndr.py

```diff
@@ -94,28 +104,28 @@ def test_ndr_without_cut(self):
         }
         # train : 300, val : 300, test : 300
         np.random.seed(1234)
-        source = self._generate_classification_dataset(config, 10)
+        source = self._generate_dataset(config, 10)
 
         result = ndr.NDR(source, working_subset='train', seed=12145)
 
-        self.assertEqual(2, len(result.get_subset("train")))
-        self.assertEqual(298, len(result.get_subset("duplicated")))
+        self.assertEqual(1, len(result.get_subset("train")))
```

Comment:
Why it was changed?

Reply:
After changing default block shape from (4,4) to (8,8), there was change in train and duplicated subset length